Jasmine/Levels/ViewControllers/GameLevelList/GameLevelListViewController.swift

```diff
@@ -30,7 +30,7 @@ class GameLevelListViewController: UIViewController {
             levelsCategoriesToggle.selectedSegmentIndex
                 = newValue ? GameLevelListViewController.builtInLevelsIndex
                            : GameLevelListViewController.customLevelsIndex
-            reloadAllData()
+            onLevelsCategoriesSwitched()
```

Comment:
Nope it still works now. `onLevelsCategoriesSwitched` calls `reloadAllData` so it's the same thing.